import/controls/permissionfilterproxymodel.cpp

```diff
@@ -0,0 +1,68 @@
+/****************************************************************************
+**
+** Copyright (C) 2020 Open Mobile Platform LLC
```

Comment:
```suggestion
** Copyright (c) 2020 Open Mobile Platform LLC.
```
Lower case c and ending with .